pace.coffee

```diff
@@ -202,7 +202,7 @@ class Bar
       @el = document.createElement 'div'
       @el.className = "pace pace-active"
 
-      document.body.className = document.body.className.replace /pace-done/g, ''
+      document.body.className = document.body.className.replace / pace-done/g, ''
```

Comment:
Good catch. Looks like there are a few possibilities I didn't consider:
```
pace-done
pace-done someclass
someclass pace-done
someclass pace-done someotherclass
```
I think `/(^pace-done ?)|( pace-done)/` does the trick for all of them? See any way to make that cleaner?